lib/draper/query_methods.rb

```diff
@@ -0,0 +1,19 @@
+require 'draper/query_methods/load_strategy'
```

Comment:
I think you can use `require_relative` here

Reply:
Good catch, thanks.